docs/source/tutorials/open_images_evaluation.ipynb

```diff
@@ -0,0 +1,444 @@
+    "5. [Prepare the ground-truth for evaluation](#Prepare-the-ground-truth-for-evaluation) (optional)\n",
+    "6. [Evaluate on a per-image granularity](#Evaluate-on-a-per-image-granularity)\n",
+    "7. [Explore](#Explore)\n",
+    "\n",
+    "Optional steps may not be necessary depending on if you have already downloaded the data or have your own model to evaluate.\n",
+    "\n",
+    "This tutorial evaluates a model on [Open Images V4](https://storage.googleapis.com/openimages/web/download_v4.html)\n",
+    "however this code supports later versions of Open Images as well. If using a newer version just make sure to\n",
+    "use the appropriate hierarchy file and class label map."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Requirements\n",
+    "\n",
+    "This notebook contains bash commands. To run it as a notebook, you must install the [Jupyter bash kernel](https://github.com/takluyver/bash_kernel) via the command below.\n",
+    "\n",
+    "Alternatively, you can just copy + paste the code blocks into your shell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install bash_kernel\n",
+    "python -m bash_kernel.install"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This workflow requires a few required python packages.\n",
+    "\n",
+    "Install the appropriate version of `tensorflow` depending on whether or not you\n",
+    "have a GPU:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install tensorflow\n",
+    "# pip install tensorflow-gpu"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Important** This notebook uses a collection of helper scripts and modules. If you downloaded this notebook from the FiftyOne docs, you will also need to download the `open_images_eval` subdirectory from\n",
+    "[fiftyone/docs/source](https://github.com/voxel51/fiftyone/tree/develop/docs/source)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the test data and ground-truth labels\n",
+    "\n",
+    "All of the data (images, metadata and annotations) can be found on the\n",
+    "[official Open Images website](https://storage.googleapis.com/openimages/web/download_v4.html).\n",
+    "\n",
+    "If you are using Open Images V4 you can use the following commands to download\n",
+    "all the necessary files."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download the data\n",
+    "\n",
+    "**WARNING** This is 36GB of data!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aws s3 --no-sign-request sync s3://open-images-dataset/test [target_dir/test]"
```

Comment:
Just replace `[target_dir/test]` with something concrete like `open-images-fiftyone/test` to make it so you can run it directly.